packages/components/src/date-time/date.js

```diff
@@ -1,150 +1,53 @@
 /**
  * External dependencies
  */
-import ReactDatePicker from "react-datepicker";
-import "react-datepicker/dist/react-datepicker.css";
-
-/**
- * WordPress dependencies
- */
-import { Component, createRef } from '@wordpress/element';
+import set from 'date-fns/set';
+import format from 'date-fns/format';
```

Comment:
Correct me if I'm wrong, but couldn't this be:

```suggestion
import { format, set } from 'date-fns';
```